greenpithumb/greenpithumb.py

```diff
@@ -68,4 +68,10 @@ def main(args):
         '--config_file',
         help='Wiring config file',
         default='wiring_config.ini')
+    parser.add_argument(
+        '-s',
+        '--sleep_windows',
+        type=str,
+        help='Sleep windows as 24-hour clock strings, with commas between each window',
```

Comment:
I think we can add more detail here. We can make the help pretty long when needed like:

>Time windows during which GreenPiThumb will not activate its pump. The windows should be in the form of a comma-separated list of time ranges in 24-hour format, such as "22:00-8:00,3:15-3:45".